### DISCUSSION TITLE
Question - digit-specific passwords

### DISCUSSION THREAD

**Discussion by SAS-1** [dani-garcia/vaultwarden#3301] [Q&A] [answered]
Hi Team

Quick question, I was reading the official Bitwarden blog today and in particular this post https://bitwarden.com/blog/filling-in-digit-specific-passwords/ 
This would be really handy for me, but it doesn't seem to be showing for my self hosted Vaultwarden server using the browser extension. 

I looked at the 2 PR's and I don't see anything needing server setup, any ideas how this works with Vaultwarden or if it does?
PR's https://github.com/bitwarden/clients/pull/3262 & https://github.com/bitwarden/clients/pull/1780

it says it was deployed in 2023.01 for the web and desktop client , I have 2023.02 installed.

Thanks

**Comment by stefan0xC** [comment, marked as answer]
It works for me and it should work for you too (because this is only dependent on the client): 

https://user-images.githubusercontent.com/509385/222817082-78f847d2-919f-4149-b962-956781846751.mp4

So you have to first open the entry, click on the eye symbol to view the password and then on the numbered list symbol to show this new feature.

**Comment by SAS-1** [comment]
thanks for the demo, the icon is not showing for me when I click view,  I will uninstall and reinstall the plugin in case there is an issue.

**Reply by stefan0xC** [reply]
Make sure your browser extension is the latest version (and you'll also probably need the permission to view the item).

It says Version: 2023.2.1 in my firefox extension (I think under Settings -> "About us").
![vaultwarden-browserextension-version](https://user-images.githubusercontent.com/509385/222820860-1916f05d-b311-4549-b270-ed76f4ba3842.png)

So I have not tested with the latest `testing` image if it still works but I think it should.
